### ISSUE TITLE
`--user` flag should not be ignored

### ISSUE THREAD

**Issue by cormacmccarthy** [skywinder/github-changelog-generator#397]
I am specifying a user using the `-u` or `--user` flag in the command line.

I expect this to result in my command line specified user being used.

The actual result is that it "detects" me as the user, and uses me instead.

The issue with this is that the repository I am trying to generate release notes for is not owned by me, I work from (and issue pull requests) from a fork of a repo owned by another "user" (which is where we manage releases from). So, when I'm trying to generate release notes, I need to hit that user instead.

```
github_changelog_generator --no-author --compare-link --github-site=https://github.mysite.com --token=$CHANGELOG_GE_TOKEN --github-api=https://github.mysite.com/api/v3/ -u=shareduser
Detected user:cormacmccarthy, project:your-project
...
:user=>"cormacmccarthy",
...
```

**Comment by cormacmccarthy** [comment]
Same thing happens if I create a `.github_changelog_generator` file and put `user=shareduser` in there.

**Comment by Turbo87** [comment]
@skywinder has the fix for this been released? I'm using v1.13.1 and I'm having the same issue.

**Comment by Turbo87** [comment]
so apparently this works if you use `-u` **and** `-p`, but not if you only use `-u`.
